### ISSUE TITLE
FreeBSD is missing in pyTooling/Common/Platform.py

### ISSUE THREAD

**Issue by yurivict** [pyTooling/pyTooling#84]

**Comment by Paebbels** [comment]
@yurivict this issue has no description ...

----------
Can you provide the values for Python's internal APIs for platform, architecture, etc.?

This article https://stackoverflow.com/questions/1854/how-to-identify-which-os-python-is-running-on/54837707#54837707 doesn't cover FreeBSD.

```python
import os
import sys
import platform
import sysconfig

print("os.name                      ",  os.name)
print("sys.platform                 ",  sys.platform)
print("platform.system()            ",  platform.system())
print("sysconfig.get_platform()     ",  sysconfig.get_platform())
print("platform.machine()           ",  platform.machine())
print("platform.architecture()      ",  platform.architecture())
```

**Comment by yurivict** [comment]
```
$ python3.9 x.py
os.name                       posix
sys.platform                  freebsd13
platform.system()             FreeBSD
sysconfig.get_platform()      freebsd-13.1-STABLE-amd64
platform.machine()            amd64
platform.architecture()       ('64bit', 'ELF')
```

**Comment by Paebbels** [comment]
Thanks. 

I assume `freebsd13` is the OS name on one side, but also a version number, right? Is it more like a category number like Windows 7, 10, 11 with several sub versions. Or a real version like Debian 11 or Ubuntu 2023.04?

If I remember correctly, macOS is also part of the BSD family. Or do you consider FreeBSD (and maybe any other BSD) been far a way from macOS? My question is: should `Platform` reflect the common root to the old days of BSD?

Are there other BSD variants it should reflect too?

**Comment by yurivict** [comment]
freebsd13 is OS name concatenated with OS version number.

> If I remember correctly, macOS is also part of the BSD family. Or do you consider FreeBSD (and maybe any other BSD) been far a way from macOS? My question is: should Platform reflect the common root to the old days of BSD?

They are very far away these days. macOS should be in a separate category.

> Are there other BSD variants it should reflect too?

The other variants are OpenBSD and NetBSD. But I don't know what are these identified as in Python - I have no access to these systems.

**Comment by Paebbels** [comment]
This is addressed in #85.

I have no option in checking it. Should I just release or do you want to try it locally on your machine?

**Comment by Paebbels** [comment]
It's merged.
